doc/source/whatsnew/v1.2.5.rst

```diff
@@ -35,7 +35,7 @@ Bug fixes
 Other
 ~~~~~
 
--
+- :meth:`DataFrame.dropna` now accepts a single column / string as ``subset`` instead of array-like
```

Comment:
Instead of "column / string", can you use "label". Also, "instead of" suggests to me it no longer accepts array-likes. I think "along with" (or something similar) would be more clear.

Reply:
Thanks, adjusted this accordingly.

Reply:
Just to be sure, you set this for 1.3 milestone, I added this to 1.2.5 whatsnew entry, should I move it to 1.3?

Reply:
This is not a regression so 1.3

Reply:
Moved.